public/omni/getting-started/how-to-install-talosctl.mdx

```diff
@@ -40,6 +41,25 @@ Users with at least the [Operator](https://omni.siderolabs.com/reference/acls#ro
 | `talosctl service <id> [stop\|start\|restart]` |
 | `talosctl shutdown`                            |
 
+Additionally, administrators (users with `os:admin` role) can run these commands through Omni:
+
+| Command                     |
+| --------------------------- |
+| `talosctl meta write`       |
+| `talosctl meta delete`      |
+| `talosctl blockdevice wipe` |
+
+
+Examples of operator level commands **blocked** when routed through Omni include:
+
+- **File & system inspection**: `talosctl ls`, `talosctl read`, `talosctl readfile`
+- **Debugging/state**: `talosctl logs`, `talosctl dmesg`, `talosctl containers`
+- **Config changes**: `talosctl apply-config`, `talosctl patch mc`, `talosctl reset`, `talosctl upgrade`
+- **Networking**: `talosctl apply-netconfig`, `talosctl get links`, `talosctl get addrs`
```

Comment:
there's no `talosctl apply-netconfig` command.
`talosctl get links` is allowed
`talosctl get addrs` doesn't exist.